src/domain/bitcoin/lightning/index.types.d.ts

```diff
@@ -79,7 +79,7 @@ type RegisterInvoiceArgs = {
 type RegisteredInvoice = {
   invoice: LnInvoice
   pubkey: Pubkey
-  descriptionHash: string // TODO: proper type
+  descriptionHash?: string // TODO: proper type
```

Comment:
Don't like having optional fields in domain types. If there is a real use-case where this is missing can we use `| null` instead of `?`

Reply:
I'm not actually sure what should be done with this type yet. this has been added as part of the lnpay feature and I just have added the ? to make the test pass. someone should look into this eventually, I'm not sure in which case we need or not the `| null`